### ISSUE TITLE
Stop spamming the terminal with exceptions

### ISSUE THREAD

**Issue by whizzer0** [neontribe/untangled-2017#88]
Make it end pls

```Everything is lava: local variable 'network_player' referenced before assignment
Traceback (most recent call last):
  File "./client.py", line 326, in run
    if network_player:
UnboundLocalError: local variable 'network_player' referenced before assignment

**Comment by Spannner** [comment]
Fixed in pull request #102

**Comment by whizzer0** [comment]
Still happens:

```
Everything is lava: 'NoneType' object is not subscriptable
Traceback (most recent call last):
  File "./client.py", line 307, in run
    new_position = bson.loads(event.msg[0])
TypeError: 'NoneType' object is not subscriptable
```

**Comment by Spannner** [comment]
Fixed in pull request #112
